Basic/ClientDriven/Assets/UI Toolkit/2DSpaceShooterUIStyleSheet.uss

```diff
@@ -0,0 +1,241 @@
+* {
```

Comment:
I see 2D space shooter in the title, I'm assuming this was copied from the other bitesize?
I'm wondering if couldn't just create ourselves a common package between bitesize projects for things like these. Were there many changes to that sheet vs the space shooter?
might be something to sync with Jil